app/assets/stylesheets/application.scss

```diff
@@ -44,6 +44,10 @@ body, html {
   line-height: 1.4;
 }
 
+.body--modal-open {
```

Comment:
Our current convention for CSS id and class names uses camelcase, let's rename this to `.bodyModalOpen`